lib/cli/src/initiate.ts

```diff
@@ -35,6 +35,7 @@ import raxGenerator from './generators/RAX';
 import { warn } from './warn';
 import { JsPackageManagerFactory, readPackageJson } from './js-package-manager';
 import { NpmOptions } from './NpmOptions';
+import { litGenerator } from './generators/LIT';
```

Comment:
For consistency sake maybe make it a default export, like the other generators?
```suggestion
import litGenerator from './generators/LIT';
import { warn } from './warn';
import { JsPackageManagerFactory, readPackageJson } from './js-package-manager';
import { NpmOptions } from './NpmOptions';
```

Reply:
I'm really not a big fan of default export and would prefer replacing all default exported generators with named ones.
Main reason: It's a real pain when you look for a `litGenerator` and end up with 15 generators 🙃 

I will use default export for consistency but in my next PR on the CLI they will be moved to named exports 😁